extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -98,6 +103,11 @@ const usGovAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.usgovcloudapi.net',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
+			azureDevOpsResource: {
+				id: 'ado',
+				endpoint: '499b84ac-1321-427f-aa17-267ca6975798',
```

Comment:
Have you verified this?

Reply:
What does it mean when the endpoint is a guid anyways?

Reply:
@aaomidi: This is the link I used to verify: https://github.com/Microsoft/AzureStorageExplorer/issues/61#issuecomment-383675754

@Charles-Gagnon: endpoint/resource Id are used to identify what to authenticate against

Reply:
Weird - so endpoint isn't necessarily an endpoint? Can we update the property name then? Looking at the code it looks like this eventually just gets passed in as the resourceId to the login request in the end - is that correct @aaomidi ?

Reply:
yeah they're technically resources. I wouldn't update it since the contract is used in SqlToolsService too. We'd have to be careful.

Reply:
We're eventually going to move away from this to scopes....so

Reply:
@joberume I think you should validate it with a US govt account. There are a few people on the team that can help you get that.

Reply:
@aaomidi As per documentation, there [isn't a dedicated US Gov Azure DevOps instance](https://docs.microsoft.com/en-us/Azure/azure-government/connect-with-azure-pipelines). The rationale behind it seems to be the government entities has to have their own instances hosted. Looks like leveraging the _same_ public endpoint is possible, albeit not the recommended path. More comprehensive list [here](https://docs.microsoft.com/en-us/azure/azure-government/compare-azure-government-global-azure). Shall I proceed in removing it?

Reply:
It won't really matter either way, maybe removing it would be the right move. Just double check that the auth code handles undefined resources fine.